src/features/personDetails/index.js

```diff
@@ -0,0 +1,47 @@
+import { useParams } from "react-router-dom";
+import { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import Loader from "../../common/Loader";
+import { Error } from "../../common/Error";
+import { selectStatus, selectDetails, selectCast, selectCrew, getDetailsForPerson } from '../personSlice';
+
+const PersonDetails = () => {
+    const secureBaseUrl = "https://image.tmdb.org/t/p/h632";
+    const dispatch = useDispatch();
+    const { id } = useParams();
+
+    useEffect(() => {
+        dispatch(getDetailsForPerson({ personId: id }));
+    }, [id, dispatch]);
+    const status = useSelector(selectStatus);
+    const details = useSelector(selectDetails);
+    const cast = useSelector(selectCast);
+    const crew = useSelector(selectCrew);
+    const castNumber = (cast.filter(({ title }) => title)).length;
+    const crewtNumber = (crew.filter(({ title }) => title)).length;
```

Comment:
`crewtNumber` - isn't it supposed to be `crewNumber`?

Reply:
unfortunately not. movies and series are together in one table

Reply:
I mean just the name of this constant. Why in `const crewtNumber` is there a "`t`" in the middle 🤔

Reply:
It's a good question, indeed it looks like a typo.